test_template.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
```

Comment:
duplicate of the tests in the github action

Reply:
Almost. Do you want to call "test_template.sh" from github actions instead?

Reply:
that would work, especially this type of code tend to rot pretty easily, so it's nice to know it's exercised